src/display/api.js

```diff
@@ -466,6 +466,14 @@ var PDFDocumentProxy = (function PDFDocumentProxyClosure() {
       return this.pdfInfo.numPages;
     },
     /**
+     * @return {string} File name of the pdf if available. If the source
+     *   is a URL, file name will be extracted from content-disposition
+     *   header.
+     */
```

Comment:
"File name of the PDF, if the PDF was fetched over a network and the Content-Disposition response header was set."